src/test/java/webdriver/DriverFactory.java

```diff
@@ -0,0 +1,35 @@
+package webdriver;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeDriverService;
+
+import java.util.concurrent.TimeUnit;
+
+public class DriverFactory {
+    private static final String CHROMEDRIVER_EXE_PATH = "src/main/resources/chromedriver.exe";
+    public static final String ONLINER_URL = "https://www.onliner.by/";
+    public static final int IMPLICITLY_WAIT = 10;
+
+    public enum BrowserType {FIREFOX, CHROME, EDGE, IE, OPERA}
+
+    public static WebDriver getDriver(BrowserType type) {
+        WebDriver driver = null;
+        switch (type) {
+            case CHROME:
+                driver = getChromeDriver();
+                break;
+        }
+        assert driver != null;
```

Comment:
instead of this i would add default section to switch and throw an exception or smth like that
        assert driver != null;   doesn't give us any additional info, it's useless for us